Mijn den haag (#46)
nl-design-system/gebruikersonderzoek: docs/onderzoek-bekijken/mijn-zaken/denhaag-mijn-omgeving.md

### Houd rekening met de wens van inwoners om veel bevestiging te krijgen

Men zoekt veel zekerheid dat uitgevoerde taken goed verwerkt zijn en vragen beantwoord zullen worden.

### Wees in de e-mailnotificatie meteen concreet over waarom men bericht ontvangt

Zet in het eerste bericht duidelijk waarom de inwoner benaderd wordt en wat hij of zij eventueel moet doen. Korte instructies in de vorm van te doorlopen stappen werken hierbij goed.

### Een duidelijk herkenbaar onderwerp in de notificatiemail is belangrijk

Zorg dat de onderwerpregel van het notificatiebericht duidelijkheid is. En dat het herkenbaar is voor de gebruiker als onderwerp dat daadwerkelijk op hen van toepassing is of kan zijn. Dit geldt ook voor de inhoud van het bericht.

### Geef simpele uitleg en escape-mogelijkheden bij online taken waarvoor meer digivaardigheid nodig is

Wanneer mensen bijvoorbeeld eigen documenten moeten uploaden, zorg dan voor een heel duidelijke toelichting en eventueel de mogelijkheid om deze op een andere manier aan te leveren (bijvoorbeeld per post).

### Onderzoek mogelijkheden om de lijst met contactmomenten eenvoudiger herkenbaar te maken

> Vooral op mobiel is het belangrijk dat de betekenis van de (waardevolle) lijst met contactmomenten sneller herkend wordt. Mogelijk is de toevoeging van enkele woorden als 'tijdlijn' of' uw eerderecontactmomenten over deze zaak' voldoende.

Reviewer: ```suggestion
Vooral op mobiel is het belangrijk dat de betekenis van de (waardevolle) lijst met contactmomenten sneller herkend wordt. Mogelijk is de toevoeging van enkele woorden als 'tijdlijn' of' uw eerdere contactmomenten over deze zaak' voldoende.
```

> Vooral op mobiel is het belangrijk dat de betekenis van de (waardevolle) lijst met contactmomenten sneller herkend wordt. Mogelijk is de toevoeging van enkele woorden als 'tijdlijn' of' uw eerderecontactmomenten over deze zaak' voldoende.

Reviewer: ```suggestion
Vooral op mobiel is het belangrijk dat de betekenis van de (waardevolle) lijst met contactmomenten sneller herkend wordt. Mogelijk is de toevoeging van enkele woorden als 'tijdlijn' of' uw eerdere contactmomenten over deze zaak' voldoende.
```

### Zelf kunnen uitvoeren van taken in de Mijn-omgeving wordt gewaardeerd

Het sluit aan bij wat men gewend is in andere on line omgevingen, zoals MijnOverheid, maar ook bijvoorbeeld webshops: snel en on line zaken regelen, waarbij je duidelijk ziet wat er gebeurt en dit ook later terug kunt vinden.

### Ervaringen met andere online omgevingen en met overheidsinstanties scheppen ook verwachtingen

Men verwacht bijvoorbeeld een bevestigings-e-mail na afronding van een taak. Dit sluit ook aan bij enig scepticisme ten aanzien van overheidsdienstverlening: men heeft soms slechte ervaringen en/of verwachtingen met betrekking tot de follow up en beperkt vertrouwen dat het 'goedkomt'.

### Notificatie via e-mail wordt als belangrijk en logisch gezien

Een e-mailnotificatie vindt men een handige reminder. Een e-mail ter bevestiging van een afgeronde taak wordt zelfs als essentieel gezien. Het wordt gewaardeerd als het onderwerp of de gevraagde actie meteen duidelijk is, evenals de deadline.

### Gebruikers vertrouwen de notificatiemail, maar zijn zich ook zeer bewust van het risico op phishing

> Vrijwel a lie gebruikers zijn zich erg bewust van de risico's op phishing bij e-mail. Ze openen de notificatiemail wel als ze de afzender herkennen en het onderwerp logischerwijs van deze afzender verwachten.

Reviewer: ```suggestion
Vrijwel alle gebruikers zijn zich erg bewust van de risico's op phishing bij e-mail. Ze openen de notificatiemail wel als ze de afzender herkennen en het onderwerp logischerwijs van deze afzender verwachten.
```

### Het uitvoeren van taken gaat overwegend soepel

> Dit geldt met name voor het doen van een beta ling. Een document uploaden vindt niet iedereen even makkelijk. Wanneer men iets moet aanleveren wil men ook graag vooraf weten wat er precies gevraagd wordt en waarom.

Reviewer: ```suggestion
Dit geldt met name voor het doen van een betaling. Een document uploaden vindt niet iedereen even makkelijk. Wanneer men iets moet aanleveren wil men ook graag vooraf weten wat er precies gevraagd wordt en waarom.
```

- Betrokkenen: Interviews en analyses door onderzoekers van gemeenten Den Haag en Utrecht. Veel andere betrokkenen hebben meegekeken. 

#### Analyse- en haalbaarheidssessie 

- Bespreking inzichten en aanbevelingen door onderzoekers en ontwerpers met de opdrachtgever. 
- Praktische en technische uitvoerbaarheid van de aanbevelingen besproken. 
- Prioritering bepaald. 
- Betrokkenen: onderzoekers en ontwerpers Den Haag en Utrecht, opdrachtgever VNG.

### Onderzoeksspecificaties

#### Methode

- Vorm: usability testing met een prototype. Respondenten kregen verschillende taken. De facilitator stelde vragen om de acties, gedachtegang en eventuele moeilijkheden beter te begrijpen. 
- Het prototype had als basis de Mijn-omgeving van DenHaag.nl (MijnDenHaag). 
- De helft testte op desktop, de andere helft op mobiel. 
- Bij de desktopversie is eye-tracking toegepast voor meer inzicht in waar de aandacht naartoe gaat. 
- Locatie: de 'Stadskamer' van de gemeente Den Haag. Er is door veel betrokkenen meegekeken via videostreaming in de fysieke meekijkruimte van het Stadhuis en via MS Teams. 

#### Doelgroep en steekproef

> - Er zijn vooral inwoners betrokken die minder don gemiddeld digivaardig en minder don gemiddeld taalvaardig zijn. Uitgangspunt was dot als zij met de Mijn-omgeving overweg kunnen, andere gebruikers dot oak kunnen. Ter vergelijking zijn er twee respondenten met een hoog opleidingsniveau betrokken.

Reviewer: ```suggestion
- Er zijn vooral inwoners betrokken die minder dan gemiddeld digivaardig en minder dan gemiddeld taalvaardig zijn. Uitgangspunt was dat als zij met de Mijn-omgeving overweg kunnen, andere gebruikers dat ook kunnen. Ter vergelijking zijn er twee respondenten met een hoog opleidingsniveau betrokken.
```

- In totaal hebben 14 respondenten deelgenomen, verdeeld over twee onderzoeksrondes.

> - Respondenten zijn deels via een wervingsbureau geworven, deels via de taalschool waar de gemeente mee samenwerkt (voor laaggeletterde inwoners) en deels via een wijkproject in de Schilderswijk (voor inzicht in de doelgroep die de Nederlandse tool minder goed beheerst).

Reviewer: ```suggestion
- Respondenten zijn deels via een wervingsbureau geworven, deels via de taalschool waar de gemeente mee samenwerkt (voor laaggeletterde inwoners) en deels via een wijkproject in de Schilderswijk (voor inzicht in de doelgroep die de Nederlandse taal minder goed beheerst).
```

- Er is gezorgd voor goede spreiding op leeftijd en geslacht.

#### Periode

- Het onderzoek vond plaats in twee rondes. Ronde 1 vond plaats tussen 16 en 19 januari 2023.  
- Ronde 2 vond plaats tussen 21 en 23 februari 2023.
- De test duurde 45 minuten per respondent.

#### Kanttekening

> Let op: de test vond niet in een 'real life setting' plaats. De werkelijkheid is nagebootst door mensen zich te laten inleven in een scenario. Dit kan ervoor zorgen dot men bepaalde zaken in de testomgeving anders opvat don wanneer men daadwerkelijk in de situatie zou zitten.

Reviewer: ```suggestion
Let op: de test vond niet in een 'real life setting' plaats. De werkelijkheid is nagebootst door mensen zich te laten inleven in een scenario. Dit kan ervoor zorgen dat men bepaalde zaken in de testomgeving anders opvat dan wanneer men daadwerkelijk in de situatie zou zitten.
```

### Testscenario's

#### Uitgangspunten

- Respondent heet 'Anne Klap' 
- Anne heeft een parkeerbon ontvangen van de gemeente, deze moet betaald warden 
- Anne heeft een zaak 'aanvraag subsidie geluidsisolatie' bij de gemeente lopen 

#### Opdrachten ronde 1

- Taak uitvoeren: Betaal de parkeerbon

> - Bekijken taken: Bekijk wat er gevraagd wordt random jouw aanvraag voor subsidie voor geluidsisolatie (er hoeft niets uitgevoerd te warden).

Reviewer: ```suggestion
- Bekijken taken: Bekijk wat er gevraagd wordt rondom jouw aanvraag voor subsidie voor geluidsisolatie (er hoeft niets uitgevoerd te worden).
```

> - Bekijken taken: Bekijk wat er gevraagd wordt random jouw aanvraag voor subsidie voor geluidsisolatie (er hoeft niets uitgevoerd te warden).

Reviewer: ```suggestion
- Bekijken taken: Bekijk wat er gevraagd wordt rondom jouw aanvraag voor subsidie voor geluidsisolatie (er hoeft niets uitgevoerd te worden).
```
